utils/auto_dockerfile_update/update_dockerfiles.py

```diff
@@ -40,13 +40,133 @@ def is_docker_file_outdated(dockerfile: Dict, latest_tag: str, last_updated: str
 
     return False
 
+def extract_current_python_version(file_path: str)->Tuple[List,str,bool]:
+    """Extract the current python version from Pipfile or the pyproject.toml.
+    
+        Args:
+            file_path (str): The file path to the Pipfile or the pyproject.toml file.
+
+        Returns:
+            Tuple[List,str, bool]: The python version in list and str.
+    """
+    python_version = ""
+    try:
+        with open(file_path, "r") as f:
+            file_content = f.read()
+        if "Pipfile" in file_path:
+            python_version = re.search(PYTHON_VERSION_PIPFILE_REGEX, file_content)
+        elif "pyproject.toml" in file_path:
+            python_version = re.search(PYTHON_VERSION_PYPROJECT_REGEX, file_content)   
+    except Exception as e:
+        print(f"{e}: Can't read file {file_path}")
+    if python_version:
+        full_str_python_version = python_version.group(0)
```

Comment:
whats the difference? Why do we need both of these

Reply:
The full_str_python_version is "python_version= "3.9"" format, we need it in order to replace it in the pipfile/pyproject file.
The string_version_number is only the version 3.9